app/assets/javascripts/maps.js

```diff
@@ -19,17 +19,7 @@ function initializeGoogleMapsAPI(){
 function initializeMaps(){
   pins = new Array();
   getCurrentLocation();
-
-  if(typeof(gon.latitude) !== 'undefined' && typeof(gon.longitude) !== 'undefined'){
-    // Geocoordinates were provided
-    initialLocation = new google.maps.LatLng(gon.latitude, gon.longitude);
-    generateMap();
-  } else if(typeof(gon.address) !== 'undefined'){
-    // Need to lookup address
-    addressLookup(gon.address);
-  } else {
-    alert("Error loading Google maps, please reload the page.");
```

Comment:
Change this to `console.log("Error loading Google maps, please reload the page.")`

Try adding `generateMap()` on a new line.